lib/pharos/autoload.rb

```diff
@@ -11,6 +11,8 @@
 autoload :K8s, 'k8s-client'
 autoload :Excon, 'excon'
 autoload :K8s, 'k8s-client'
+autoload :Open3, 'open3'
```

Comment:
Just a note that this autoload thing is starting to look a bit monster-ish... we probably should look how to make autoloading better.